contracts/asset-proxy/contracts/src/ERC1155Proxy.sol

```diff
@@ -0,0 +1,268 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity 0.4.24;
+
+import "./MixinAuthorizable.sol";
+
+
+contract ERC1155Proxy is
+    MixinAuthorizable
+{
+
+    // Id of this proxy.
+    bytes4 constant internal PROXY_ID = bytes4(keccak256("ERC1155Token(address,uint256[],uint256[],bytes)"));
+
+    function () 
+        external
+    {
+        // Input calldata to this function is encoded as follows:
+        //                      -- TABLE #1 --
+        // | Area     | Offset (**) | Length      | Contents                        |
+        // |----------|-------------|-------------|---------------------------------|
+        // | Header   | 0           | 4           | function selector               |
+        // | Params   |             | 4 * 32      | function parameters:            |
+        // |          | 4           |             |   1. offset to assetData (*)    |
+        // |          | 36          |             |   2. from                       |
+        // |          | 68          |             |   3. to                         |
+        // |          | 100         |             |   4. amount                     |
+        // | Data     |             |             | assetData:                      |
+        // |          | 132         | 32          | assetData Length                |
+        // |          | 164         | (see below) | assetData Contents              |
+        //
+        //
+        // Asset data is encoded as follows:
+        //                      -- TABLE #2 --
+        // | Area     | Offset      | Length  | Contents                            |
+        // |----------|-------------|---------|-------------------------------------|
+        // | Header   | 0           | 4       | assetProxyId                        |
+        // | Params   |             | 4 * 32  | function parameters:                |
+        // |          | 4           |         |   1. address of ERC1155 contract    |
+        // |          | 36          |         |   2. offset to tokenIds (*)         |
```

Comment:
I feel like these param names should be `ids`, `values`, and `data` to be consistent with the naming of the actual ERC1155 transfer params.

Reply:
Meh, actually we aren't really consistent in the other proxies either.